window.py

```diff
@@ -61,10 +61,6 @@ def read(self, event):
         if self.table_panel is not None and self.cont_measurement:
             self.table_panel.read_last(event)
 
-    def update(self):
-        self.Bind(wx.EVT_TIMER, self.check)
-        self.timer.StartOnce(10)
-

```

Comment:
toto sa preco vymazalo

Reply:
ja som si to pomylil s methodou Update ktoru ma wxpython na aktualizaciu okna

Reply:
som mala pocit ze sa to uz nikde nevola